Grammar/python.gram

```diff
@@ -128,6 +128,8 @@ simple_stmt[stmt_ty] (memo):
 compound_stmt[stmt_ty]:
     | &('def' | '@' | ASYNC) function_def
     | &'if' if_stmt
+    | &'elif' nomatch_elif
```

Comment:
Unfortunately I'm not convinced about this, all of these should be in an "invalid_" rule and never inlined in the correct rule.